### ISSUE TITLE
MongoDB.shell fails to install, even as admin

### ISSUE THREAD

**Issue by aslze** [microsoft/winget-pkgs#28462]
# Brief description of your issue

MongoDB.shell fails to install, even as admin.

# Steps to reproduce

In a console, even run as administrator:

```
winget install mongodb.shell
```

# Expected behavior

Installation should succeed.

# Actual behavior

The program says:

```
Installer failed with exit code: 1603
```

The log file produced contains this:

```
MSI (s) (A8:14) [19:30:01:127]: Product: MongoDB 4.2.8 2008R2Plus SSL (64 bit) -- Configuration failed.

MSI (s) (A8:14) [19:30:01:127]: Windows Installer volvió a configurar el producto. Nombre del producto: MongoDB 4.2.8 2008R2Plus SSL (64 bit). Versión del producto: 4.2.8. Idioma del producto: 1033. Fabricante: MongoDB Inc.. Resultado de la nueva configuración: 1603.
```

# Environment

```
[winget --info]
Windows Package Manager v0.2.2521 Preview
Windows: Windows.Desktop v10.0.19041.508
Package: Microsoft.DesktopAppInstaller v1.11.2521.0
```

Well, previously I installed **MongoDB.server**, which reported *success*:

```
winget install mongodb.server
```

But in fact I can't find it installed. Under `Program Files\MongoDB\Server\4.2` I can see some text files (README, etc), but no executables. (?). Can this be related to the failure of installing the shell?

**Comment by jamesbroadhead** [comment]
I believe that this has been fixed since #19881 with the switch to the new MongoDB shell, mongosh. 

The following works for me: 
```
winget install mongodb.shell
C:\Users\jbo\AppData\Local\Programs\mongosh\mongosh.exe
```

**Comment by denelon** [comment]
Thanks for letting us know.
